### DISCUSSION TITLE
How to configure the  clusterNodes configuration when elasticsearch ips are multi

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [apache/skywalking#11787] [Q&A]
clusterNodes: ${SW_STORAGE_ES_CLUSTER_NODES:localhost:9200}
I have a question,  how to configure the  es clusterNodes configuration when elasticsearch ips are multi ,does it need add "" ,for example:
when there're 3 es ips,  107.200.60.1, 107.200.60.2, 107.200.60.3,
 should I configure clusterNodes like this:  clusterNodes: "107.200.60.1:9200, 107.200.60.2:9200, 107.200.60.3:9200" ? 
 or like this(without ""): clusterNodes: 107.200.60.1:9200, 107.200.60.2:9200, 107.200.60.3:9200   , or else ?
 how should I configure when there 3 ips?

**Comment by wu-sheng** [comment]
Use `ASK AI` on the website, such as `How to config IPs of elasticsearch cluster?`
